tests/constants/networks.js

```diff
@@ -1,6 +1,5 @@
 /* global Cypress */
-// import networks from '@network/configuration/networks';
-import networks from '@network/configuration/networks';
+import networks from '../../src/modules/network/configuration/networks';
```

Comment:
is the alias path working here like below?
```suggestion
import networks from '@network/configuration/networks';
```

Reply:
nope it doesn't I have also tried adding the webpack plugin that enables cypress to extend its build configurations to our own webpack config but that still didn't work.